seagull/board.py

```diff
@@ -97,3 +97,83 @@ def view(self, figsize=(5, 5)) -> Tuple[Figure, AxesImage]:
         im = ax.imshow(self.state, cmap=plt.cm.binary, interpolation="nearest")
         im.set_clim(-0.05, 1)
         return fig, im
+
+
+class MultiStateBoard:
+    """
+    Represents a multistate environment where the lifeforms can grow and
+    evolve
```

Comment:
Once we've settled on the setup, I'd highly recommend elaborating further in the docstring how the `MultiStateBoard` "works"! :smile:

Reply:
Of course :)